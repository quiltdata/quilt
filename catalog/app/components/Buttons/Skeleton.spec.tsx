import * as React from 'react'
import renderer from 'react-test-renderer'

import * as Buttons from './'

describe('components/Buttons/Skeleton', () => {
  it('render medium by default', () => {
    const tree = renderer.create(<Buttons.Skeleton />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render small', () => {
    const tree = renderer.create(<Buttons.Skeleton size="small" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render large', () => {
    const tree = renderer.create(<Buttons.Skeleton size="large" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
