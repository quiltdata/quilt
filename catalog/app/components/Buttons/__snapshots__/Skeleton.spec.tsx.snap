// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Buttons/Skeleton render large 1`] = `
<div
  className="MuiBox-root MuiBox-root-10 makeStyles-root-1 makeStyles-large-4 makeStyles-root-5 makeStyles-animate-7"
/>
`;

exports[`components/Buttons/Skeleton render medium by default 1`] = `
<div
  className="MuiBox-root MuiBox-root-8 makeStyles-root-1 makeStyles-medium-3 makeStyles-root-5 makeStyles-animate-7"
/>
`;

exports[`components/Buttons/Skeleton render small 1`] = `
<div
  className="MuiBox-root MuiBox-root-9 makeStyles-root-1 makeStyles-small-2 makeStyles-root-5 makeStyles-animate-7"
/>
`;
